tests/consensus_tests/test_custom_sharding.py

```
@@ -366,3 +366,51 @@ def try_search_random(peer_url: str, cancel: threading.Event):
         time.sleep(0.05)
 
     return True
+
+def test_no_shards_payload_index(tmp_path: pathlib.Path):
+    """
+    Test that only payload index creation is allowed on custom sharded collections without shard keys.
+    """
+    assert_project_root()
+
+    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, N_PEERS)
```

Ruff (0.14.10)

Unpacked variable `peer_dirs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)